apertium/adapt_docx.cc

```diff
@@ -233,7 +234,7 @@ Paragraph::Paragraph(xmlNode *_root)
 void showParagraphs(xmlNode *root)
 {
 	vector<xmlNode*> pp = Walker::findAll(root, "p", true);
-	cout << pp.size() << " paragraphs found" << endl;
+	cout << pp.size() << " " << I18n(APER_I18N_DATA, "apertium").format("paragraphs_found") << endl;
```

Comment:
shouldn't `pp.size()` be an argument to the format string?

Reply:
yes it should.
thanks for notice.